project1/report.Rmd

```diff
@@ -76,5 +76,52 @@ data[, serum_cat := cut(serum_bilirubin, c(0, 1.1, 3.3, max(serum_bilirubin)))]
 # Code
 
 ```{r ref.label = knitr::all_labels(), echo = TRUE, eval = FALSE}
-# this R markdown chunk generates a code appendix
+# Reading in data
+data <- read.table("/Users/sktt1teddyfakerkhan/OneDrive - Emory University/bios521/project1_data-1.csv", 

```

Comment:
Would you mind changing this to just `read.table("data.csv"`? Since we have a R project in the folder already, we will not need to use full filepaths. The "data.csv" in this git repo already has the data, so when someone opens this repo from the R project, the home directory will be set to this repo and all they have to do is run `read.table("data.csv"`. This is more reproducible than a local filepath and will allow all group members to run this code.